Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -634,12 +641,22 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
     }
 
     private func configureCell(cell: ThumbnailCell, forSuggestedSite site: SuggestedSite) {
-        cell.textLabel.text = site.title.isEmpty ? NSURL(string: site.url)?.normalizedHostAndPath() : site.title
+        let title = site.title.isEmpty ? NSURL(string: site.url)?.normalizedHostAndPath() : site.title
+        cell.textLabel.text = title
         cell.imageWrapper.backgroundColor = site.backgroundColor
         cell.imageView.contentMode = .ScaleAspectFit
         cell.imageView.layer.minificationFilter = kCAFilterTrilinear
         cell.accessibilityLabel = cell.textLabel.text
 
+        if let title = title {
+            let removeButtonAccessibilityLabel =
+                String(format:
+                    NSLocalizedString("TopSites.RemovePage.Button",
```

Comment:
Should probably factor this out since it's the exact same string/ID. One reason to use `Strings.swift` :)